javascriptv3/example_code/sns/actions/check-if-phone-number-is-opted-out.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fileURLToPath } from "url";
+
+// snippet-start:[sns.JavaScript.SMS.checkIfPhoneNumberIsOptedOutV3]
+import { CheckIfPhoneNumberIsOptedOutCommand } from "@aws-sdk/client-sns";
+
+import { snsClient } from "../libs/snsClient.js";
+
+export const checkIfPhoneNumberIsOptedOut = async (
+  phoneNumber = "5555555555"
+) => {
+  const command = new CheckIfPhoneNumberIsOptedOutCommand({
+    phoneNumber,
+  });
+
+  const response = await snsClient.send(command);
+  console.log(response);
+  // {
+  //   '$metadata': {
+  //     httpStatusCode: 200,
+  //     requestId: '3341c28a-cdc8-5b39-a3ee-9fb0ee125732',
+  //     extendedRequestId: undefined,
+  //     cfId: undefined,
+  //     attempts: 1,
+  //     totalRetryDelay: 0
+  //   },
+  //   isOptedOut: false
+  // }
```

Comment:
I don't quite understand this commented out block.

Reply:
It's the output from the SDK call.